docker/api/requirements-dev.txt

```diff
@@ -1,3 +1,8 @@
 -r requirements-tests.txt
 pycodestyle==2.8.0
 pylint==2.12.2
+pytest==6.2.5
+pytest-asyncio==0.16.0
+pytest-dependency==0.5.1
+pytest-order==1.0.1
+httpx==0.23.3
```

Comment:
These should be added to `requirements-tests.txt` as these packages are specific for testing.

Reply:
OK